src/sentry/static/sentry/app/views/performance/transactionSummary/index.tsx

```diff
@@ -139,6 +139,12 @@ class TransactionSummary extends React.Component<Props, State> {
         kind: 'function',
         function: ['tpm', '', undefined],
       },
+      {
+        // This should be changed to an aggregate function when introduced
+        // to Discover
+        kind: 'field',
```

Comment:
Is there a problem with setting this to `function` without adding it to discover?

Reply:
Yeah if I set this to 'function' it needs to be exposed in the aggregations in fields.tsx which exposes it to Discover

Reply:
Okay if that's the case, let's use a cast in the short term like this

```javascript
{
  kind: 'function',
  function: ['user_misery_prototype' as AggregationKey, threshold, undefined],
},
```

to keep the `kind` correct.